Makefile

```diff
@@ -325,7 +325,11 @@ type-gen:
 method-gen:
 	(cd ./lotuspond/front/src/chain && go run ./methodgen.go)
 
-gen: type-gen method-gen
+api-gen:
+	go run ./gen/api > api/apistruct/struct.go
+	goimports -w api/apistruct
+	goimports -w api/apistruct
```

Comment:
why twice?

Reply:
Apparently calling it once won't fully fix imports